CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Add support for building Android App Bundles (AABs)
 - Migrate `PatientsScreen` to Mobius
 - Migrated `SyncIndicatorController` to Mobius
+- Use local flag for patient delete feature
```

Comment:
Since this is the same release in which the "Delete patient" feature was added, I think we can skip this line.

Reply:
Should we even add the delete patients as part of the feature since the user cannot actually use this and we cannot enable it remotely? 🤔

Reply:
That's a good point. Let's remove this.